text/0069-api-v2.md

```diff
@@ -23,17 +23,17 @@ The `API V2` is enabled by a switch on PD. Since it changes the storage encode,
 
 Once the `API V2` is enabled, the key will be starting with either:
 
-1. `m` or `t`: TxnKV key. Used by TiDB.
-2. `k{keyspace prefix id}x`: TxnKV key.
-3. `k{keyspace prefix id}r`: RawKV key.
+1. `m` and `t`: TxnKV key that will only be used by TiDB.
+2. `x{keyspace prefix id}`: TxnKV key.
+3. `r{keyspace prefix id}`: RawKV key.
```

Comment:
could you elaborate on why this is better in the RFC?

Reply:
done